scrapers.py

```diff
@@ -478,10 +479,39 @@ def __init__(self, headless: bool = True) -> None:
         super().__init__(headless=headless)
         self.homepage = "https://judicialrecords.wilco.org/PublicAccess/default.aspx"
 
+    def fetch_parsed_case(self, case_id: str) -> Tuple[str, str]:
+        query_result = self.query_case_id(case_id)
+        if query_result is None:
+            return None
+        result_soup, register_soup = query_result
+
+        register_url = case_search.get_register_url(result_soup)
+        status, type = case_search.get_status_and_type(result_soup)
+
+        if status.lower() not in hearing.statuses_map:
+            load_dotenv()
+            if os.getenv("LOCAL_DEV") != "true":
+                log_and_email(
+                    f"Case {case_id} has status '{status}', which is not in our list of known statuses.",
+                    "Found Unknown Status",
+                    error=True,
+                )
+            else:
+                logger.info(
+                    f"Case {case_id} has status '{status}', which is not in our list of known statuses."
+                )
+
+        parser = hearing.WilliamsonParser()
+        return parser.make_parsed_case(
+            soup=register_soup, status=status, type=type, register_url=register_url
+        )
+
+
+
 
 SCRAPER_NAMES = {
```

Comment:
I think this change is why your tests are all failing on CI. The CI can only load TestScraper, not the other scrapers, because it's can't load ChromeDriver. But this change causes the CI to make instances of all four scraper classes whenever it loads scrapers.py.